### PR TITLE
Diraer names

### PR DESCRIPTION
**Pull request by lewisjared**
# Pull request

Making the conversion of the direct aerosols more consistent. Only the industrial contributions were being handled

- [x] Tests added
- [x] Description in ``CHANGELOG.rst`` added

### CODE REVIEW

File: tests/test_definitions.py [hunk 1]
```diff
@@ -66,6 +66,83 @@ def test_convert_magicc7_to_magicc6_variables(magicc7, magicc6):
     )
 
 
+@pytest.mark.parametrize("suffix", [
+    "RF",
+    "ERF"
+])
+@pytest.mark.parametrize("prefix", [
```
Comment (znicholls) [inline tests/test_definitions.py:73]:
```suggestion
@pytest.mark.parametrize("suffix_part_1", [
```

It's not really a prefix if it comes after the variable...

File: tests/test_definitions.py [hunk 2]
```diff
@@ -153,16 +233,16 @@ def test_convert_magicc7_to_openscm_variables(magicc7, openscm):
         ("CO2I_EMIS", "Emissions|CO2|MAGICC Fossil and Industrial"),
         ("CH3CCL3_EMIS", "Emissions|CH3CCl3"),
         ("CCL4_INVERSE_EMIS", "Inverse Emissions|CCl4"),
-        ("CO2_EMIS", "Emissions|CO2"),
-        ("CH4_EMIS", "Emissions|CH4"),
-        ("N2O_EMIS", "Emissions|N2O"),
+        ("CO2T_EMIS", "Emissions|CO2"),
+        ("CH4T_EMIS", "Emissions|CH4"),
+        ("N2OT_EMIS", "Emissions|N2O"),
```
Comment (znicholls) [inline tests/test_definitions.py:238]: `CO2_EMIS`, `CH4_EMIS` and `N2O_EMIS` all need to work too from memory...

File: tests/test_definitions.py [hunk 3]
```diff
@@ -153,16 +233,16 @@ def test_convert_magicc7_to_openscm_variables(magicc7, openscm):
...
         ("CO2PF_EMIS", "Land to Air Flux|CO2|MAGICC Permafrost"),
         ("SURFACE_TEMP", "Surface Temperature"),
         ("CC4F8_CONC", "Atmospheric Concentrations|cC4F8"),
         ("AIR_CIRRUS_RF", "Radiative Forcing|Aviation|Cirrus"),
         ("AIR_CONTRAIL_RF", "Radiative Forcing|Aviation|Contrail"),
         ("AIR_H2O_RF", "Radiative Forcing|Aviation|H2O"),
-        ("CH4_ERF", "Effective Radiative Forcing|CH4"),
+        ("CH4T_ERF", "Effective Radiative Forcing|CH4"),
```
Comment (znicholls) [inline tests/test_definitions.py:245]: As above, CH4_ERF should work too

File: tests/test_definitions.py [hunk 4]
```diff
@@ -66,6 +66,83 @@ def test_convert_magicc7_to_magicc6_variables(magicc7, magicc6):
...
+        assert openscm_var.startswith("Atmospheric Concentrations")
+    elif suffix == "OT":
+        assert openscm_var.startswith("Optical Thickness")
+
+    if prefix == "I":
+        assert openscm_var.endswith("MAGICC Fossil and Industrial")
+    elif prefix == "B":
+        assert openscm_var.endswith("MAGICC AFOLU")
+    else:
+        assert openscm_var.endswith(variable)
```
Comment (znicholls) [inline tests/test_definitions.py:143]: Can we also add, `assert "|{}|".format(variable) in openscm_var`?

**Comment by lewisjared** [comment]
This was more complicated than I thought...

**Comment by znicholls** [comment]
> This was more complicated than I thought...

changing everything to expect 'T' at the end wasn't exactly the easy route...

**Comment by lewisjared** [comment]
> easy route...
Nope not at all...
I backed that out now so that the default total is without the trailing T to keep the readers/writers happy.

File: tests/test_core.py [hunk 5]
```diff
@@ -1032,8 +1032,8 @@ def test_persistant_state_integration(package, rf_method):
                 "Emissions|CO2|MAGICC Fossil and Industrial",
                 "Emissions|CO2|MAGICC AFOLU",
                 "Emissions|CH4|MAGICC Fossil and Industrial",
-                "Emissions|BC|MAGICC Fossil and Industrial",
-                "Emissions|SOx|MAGICC AFOLU",
+                "Emissions|Aerosols|Direct Effect|BC|MAGICC Fossil and Industrial",
+                "Emissions|Aerosols|Direct Effect|SOx|MAGICC AFOLU",
```
Comment (znicholls) [inline tests/test_core.py:1036]:
```suggestion
                "Emissions|SOx|MAGICC AFOLU",
```

Direct effect only applies to radiative forcing, everything else shouldn't change

File: tests/test_definitions.py [hunk 6]
```diff
@@ -138,6 +204,11 @@ def test_convert_magicc7_to_magicc6_variables(magicc7, magicc6):
         ),
         ("TOTAL_INCLVOLCANIC_ERF", "Effective Radiative Forcing"),
         ("SLR_TOT", "Sea Level Rise"),
+        ("CO2T_EMIS", "Emissions|CO2"),
+        ("CH4T_RF", "Radiative Forcing|CH4"),
+        ("CH4_RF", "Radiative Forcing|CH4"),
+        ("NOX_EMIS", "Emissions|Aerosols|Direct Effect|NOx"),
+        ("NOXT_EMIS", "Emissions|Aerosols|Direct Effect|NOx"),
```
Comment (znicholls) [inline tests/test_definitions.py:211]:
```suggestion
        ("NOXT_EMIS", "Emissions|NOx"),
```

File: tests/test_io.py [hunk 7]
```diff
@@ -333,13 +333,14 @@ def test_load_ot():
...
-        mdata["variable"] == "Optical Thickness|NOx|MAGICC Fossil and Industrial"
+        mdata["variable"]
+        == "Optical Thickness|Aerosols|Direct Effect|NOx|MAGICC Fossil and Industrial"
     ).all()
 
     assert_mdata_value(
         mdata,
         0.00668115649,
-        variable="Optical Thickness|NOx|MAGICC Fossil and Industrial",
+        variable="Optical Thickness|Aerosols|Direct Effect|NOx|MAGICC Fossil and Industrial",
```
Comment (znicholls) [inline tests/test_io.py:343]:
```suggestion
        variable="Optical Thickness|NOx|MAGICC Fossil and Industrial",
```

as above

File: pymagicc/definitions/__init__.py [hunk 8]
```diff
@@ -416,11 +423,27 @@ def get_openscm_replacement(in_var):
     else:
         # these come from MAGICC's output
         one_way_replacements = {
-            "CO2T_EMIS": "Emissions|CO2",
-            "CH4T_EMIS": "Emissions|CH4",
-            "N2OT_EMIS": "Emissions|N2O",
             "SURFACE_TEMP_SUBANNUAL": "Surface Temperature",
         }
+
+        # Replace the XXX_YYY with XXXT_YYY variables
```
Comment (znicholls) [inline pymagicc/definitions/__init__.py:429]: Old comment?

Comment (znicholls) [reply pymagicc/definitions/__init__.py:429]:
```suggestion
```

File: pymagicc/io.py [hunk 9]
```diff
@@ -2659,11 +2662,14 @@ def _write_variable_datablock_concentrations(self, output, data_block, units_lev
     def _reorder_data_block_and_get_units(self, data_block, units_level):
         col_order, rename_col = self._get_col_order_rename_col()
 
-        data_block = data_block[col_order]
+        metadata = data_block.columns.to_frame().reset_index(drop=True)
+        metadata.columns = metadata.iloc[0]
 
-        units = data_block.columns.get_level_values(units_level)
-        data_block.columns = data_block.columns.droplevel(units_level)
+        # Drop the T from the total variables
```
Comment (znicholls) [inline pymagicc/io.py:2668]:
```suggestion
```

**Pull request merged**